t/ui/18-tests-details.t

```diff
@@ -457,6 +457,19 @@ subtest 'test candidate list' => sub {
     );
     test_with_error(0, 0, ['sudo-passwordprompt', 'some-other-tag'],
         \%expected_candidates, 'needles appear twice, each time under different tag');
+
+    $driver->get('/tests/99946#step/installer_timezone/1');
+    disable_bootstrap_animations;    # does not seem to cover everything so we still need wait_for_... functions
+    wait_for_ajax(msg => 'step preview');
```

Comment:
we have `wait_for_ajax_and_animations` :)

Reply:
I'll change it.